src/gui/properties/trackerlistwidget.cpp

```diff
@@ -476,13 +493,17 @@ void TrackerListWidget::loadTrackers()
         item->setData(COL_LEECHES, Qt::UserRole, prettyCount(leechesMax));
         item->setData(COL_TIMES_DOWNLOADED, Qt::UserRole, prettyCount(downloadedMax));
         item->setData(COL_MSG, Qt::UserRole, entry.message);
+        item->setData(COL_NEXT_ANNOUNCE, Qt::UserRole, QLocale().toString(nextAnnounceTime, QLocale::ShortFormat));
+        item->setData(COL_MIN_ANNOUNCE, Qt::UserRole, QLocale().toString(minAnnounceTime, QLocale::ShortFormat));
         if (!item->isExpanded())
         {
             item->setText(COL_PEERS, item->data(COL_PEERS, Qt::UserRole).toString());
             item->setText(COL_SEEDS, item->data(COL_SEEDS, Qt::UserRole).toString());
             item->setText(COL_LEECHES, item->data(COL_LEECHES, Qt::UserRole).toString());
             item->setText(COL_TIMES_DOWNLOADED, item->data(COL_TIMES_DOWNLOADED, Qt::UserRole).toString());
             item->setText(COL_MSG, item->data(COL_MSG, Qt::UserRole).toString());
+            item->setText(COL_NEXT_ANNOUNCE, item->data(COL_NEXT_ANNOUNCE, Qt::UserRole).toString());
+            item->setText(COL_MIN_ANNOUNCE, item->data(COL_MIN_ANNOUNCE, Qt::UserRole).toString());
```

Comment:
Have you considered using the format of `Utils::Misc::userFriendlyDuration()` instead?

Reply:
`Utils::Misc::userFriendlyDuration()` is for representing duration, not datetime (timepoint).

Reply:
Just to clarify. I suppose user is more interested in when the next announce is going to happen and showing 5mins (later) is intuitive than hh:mm:ss format, at least for myself.